src/components/modeler/Modeler.vue

```diff
@@ -539,7 +550,20 @@ export default {
         definition.set('name', '');
       }
 
-      store.commit('addNode', new Node(type, definition, diagram));
+      const node = this.getNodeType(type, definition, diagram);
+
+      store.commit('addNode', node);
+    },
+    getNodeType(type, definition, diagram) {
```

Comment:
Do you think a name like `getNode` or `createNode` would read better here? I saw "type" and my brain assumed this is going to return the node type, like the string `"processmaker-modeler-task"`.

Reply:
I started with getNode, but had some doubts. Can switch it.

Reply:
Maybe `createNodeFromType`?